log/log.go

```diff
@@ -41,31 +40,51 @@ var levelPrefix = [...]string{
 var (
 	// Level stores the current logging level.
 	Level = LevelInfo
-	// SysLogger is a syslog Writer to be used if not nil.
-	SysLogger *syslog.Writer
+	// outputInterface stores a SetLogger() parameter.
+	outputInterface Interface
```

Comment:
I would still call this `sysLogger` or if you *really* want to be generic, just `output`.